### PR TITLE
Remove references to senofi in the Ansible properties

### PR DESCRIPTION
**Pull request by yzhivkov**

### CODE REVIEW

File: ansible/values.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ fabric_channels:
 
 ######## Templated variables with default values, DO NOT adjust unless in advanced mode ###########
 cloud_provider: aws
-console_image: ghcr.io/senofi/fabric-console
+console_image: ghcr.io/openidl-org/fabric-console
```
Comment (yzhivkov) [inline ansible/values.yml]: @TsvetanG we may need to leave this in the Senofi repo for the time being

File: ansible/roles/git-config/tasks/push.yml
Comment (TsvetanG) [inline ansible/roles/git-config/tasks/push.yml:1]: @yzhivkov the value of the email and name should be extracted as variable and the defaulted in default.yml

File: ansible/roles/openidl-secrets/defaults/main.yml
Comment (TsvetanG) [inline ansible/roles/openidl-secrets/defaults/main.yml:1]: @yzhivkov the gitrepo branch operator-develop should change as it is senofi git specific.

File: ansible/roles/openidl-apps/defaults/main.yml
Comment (TsvetanG) [inline ansible/roles/openidl-apps/defaults/main.yml:1]: @yzhivkov the apps_image_tag can be extracted in the values.yml template file and defaulted to something (i.e. latest or leave empty if it takes latest).

File: ansible/values.yml (hunk 1 above)
Comment (TsvetanG) [reply ansible/values.yml]: yes for now I agree.  We need to have a chat on that with Sean. We want to contribute the vault integration to the fabric console project in the future.

File: ansible/roles/git-config/tasks/push.yml
Comment (yzhivkov) [reply ansible/roles/git-config/tasks/push.yml:1]: Extracted in cicd_git_email and cicd_git_name vars

File: ansible/roles/openidl-secrets/defaults/main.yml
Comment (yzhivkov) [reply ansible/roles/openidl-secrets/defaults/main.yml:1]: Defaulted to develop

File: ansible/roles/openidl-apps/defaults/main.yml
Comment (yzhivkov) [reply ansible/roles/openidl-apps/defaults/main.yml:1]: defaulted to 'latest' for now

**Review by TsvetanG — APPROVED** [review]

**Pull request merged**